recipes/h2/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "h2" %}
+{% set version = "3.0.1" %}
+{% set sha256 = "b2962f883fa392a23cbfcc4ad03c335bcc661be0cf9627657b589f0df2206e64" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - hyperframe    >=5.1.0,<6
+    - hpack         >=2.3,<4
+
+test:
+  imports:
+    - test
+
+about:
+  home: http://python-hyper.org/h2/
+  license: MIT
+  summary: 'HTTP/2 State-Machine based protocol implementation'
+  dev_url: https://github.com/python-hyper/hyper-h2
+
+extra:
+  recipe-maintainers:
+    - primozgodec
+    - astaric
```

Comment:
@astaric are you ok with being a maintainer?

Reply:
Sure